### PR TITLE
Delete extra "is" in specification doc

### PR DESCRIPTION
**Pull request by ajcwebdev**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,24 +1 @@
...
-## Maintainers
-
-The current maintainers (people who can merge pull requests) are:
-
-* Jorge Vicente Cantero - [@jvican](https://github.com/jvican)
-* Justin Kaeser - [@jastice](https://github.com/jastice)
-* Łukasz Wawrzyk - [@lukaszwawrzyk](https://github.com/lukaszwawrzyk)
-* Ólafur Páll Geirsson - [@olafurpg](https://github.com/olafurpg)
-* Tomasz Pasternak - [@tpasternak](https://github.com/tpasternak)
+### ![bsp logo](resources/buildServerProtocol64.svg) Build Server Protocol
```
Comment (jastice) [inline readme.md:1]: I don't think this is intentional?

**Review by jastice — CHANGES_REQUESTED** [review]
Thanks! Looks like you changed readme.txt inadvertantly?

File: readme.md (hunk 1 above)
Comment (ajcwebdev) [reply readme.md:1]: Ahh yeah my mistake, I was playing around after making the pull request and didn't think that would be included. Kind of a noob with this type of stuff still. How would you like me to resolve this? I can change the readme back or close this out and open a new one, I dunno is there's a fancier git way to do this.

Comment (jastice) [reply readme.md:1]: You can just change it and push, and I'll squash on merge. Or you can reset the branch and force-push to get rid of the bad commits.

Comment (ajcwebdev) [reply readme.md]: Okay, it should be good to go now. Thanks for the help and good luck with this super cool project!

**Review by jastice — APPROVED** [review]

**Pull request merged**